cli/src/deno.rs

```diff
@@ -25,8 +32,19 @@ use crate::commands::extensions::state::ExtensionState;
 use crate::commands::extensions::{api, extension};
 use crate::commands::{CommandResult, ExitCode};
 
-/// Load Phylum API for module injection.
-const EXTENSION_API: &str = include_str!("../../extensions/phylum.ts");
+/// Internal extension module that creates global Phylum object.
+const EXTENSION_API: &[ExtensionFileSource] = &include_js_files!(
+    phylum_api
+    dir "js",
+    "api.js",
+    "api_version.js",
+    "main.js");
+
+/// Importable "phylum" module for backwards compatibility.
+const PHYLUM_MODULE: &str = "
+export const PhylumApi = Phylum;
+export const ApiVersion = Phylum.ApiVersion;
+";
```

Comment:
Even with this backwards compatibility, should the documentation be updated to match the _current_ method of using the global `Phylum` object directly? If so, it looks like `PhylumApi` shows up in these doc files:

* `docs/extensions/extension_example.md`
* `docs/extensions/extension_manifest.md`
* `docs/extensions/extension_rest_api.md`
* `docs/extensions/extension_sandboxing.md`